aspnetcore/fundamentals/host/generic-host.md

```diff
@@ -1,14 +1,15 @@
 ---
-title: .NET Generic Host
+title: ASP.NET Core generic host
 author: guardrex
-description: Learn about the Generic Host in .NET, which is responsible for app startup and lifetime management.
+description: Learn about the generic host in .NET, which is responsible for app startup and lifetime management.
 monikerRange: '>= aspnetcore-2.1'
 ms.author: riande
-ms.custom: mvc
+ms.custom: mvc,seodec18
 ms.date: 11/28/2018
 uid: fundamentals/host/generic-host
 ---
-# .NET Generic Host
+
+# ASP.NET Core generic host
```

Comment:
Proper noun ..... sorry for the spam. I didn't think there were that many in titles. 😅